guides/release/components/conditional-content.md

```diff
@@ -11,6 +11,13 @@ There are 2 styles of `if`: **block** and **inline**.
 </div>
 ```
 
+Additionally, you concatenate a conditional like this: 
```

Comment:
```suggestion
Additionally, you can use template helpers like `concat` within a conditional. For the example below, if `@color` has a truthy value, such as `'navy'`, the div classes will include `badge-navy`:
```

Thanks for the PR! What do you think about these suggestions? I think it's good to tie this into the general approach of helpers. Helpers have not been introduced yet at this point in the guides, but there's a section dedicated to them within the same subheading. Possibly the "template helpers" text should be a link.

Reply:
One more thought, maybe this should go at the bottom? It could be a new section, "Combining if with other helpers." In looking at the full article, it seems like we should stick to the basics right at the very top of the article. I'm open to whatever you think is best!